### PR TITLE
chore: add migration guide link to homepage

### PR DESCRIPTION
**Pull request by Foxandxss**

### CODE REVIEW

File: misc/demo/index.html [hunk 1]
```diff
@@ -179,6 +179,9 @@
                        <pre><code>angular.module('myModule', ['ui.bootstrap']);</code></pre>
                     </p>
                     <p>You can fork one of the plunkers from this page to see a working example of what is described here.</p>
+                    <h3>Migration to prefixes</h3>
+                    <p>Since version 0.14.0 we started to prefix all our components. If you come from ui-bootstrap 0.13.4 or earlier,
```
Comment (wesleycho) [inline misc/demo/index.html]: Maybe `If you are upgrading from ui-bootstrap 0.13.4 or earlier,` instead?

**Comment by wesleycho** [comment]
Excepting that awkward bit, this PR LGTM.

**Comment by Foxandxss** [comment]
Done.

**Comment by wesleycho** [comment]
LGTM

**Comment by Foxandxss** [comment]
I need to retrofit this to all 0.14 versions too.

**Pull request closed without merging**